sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -40,6 +42,8 @@ public class HttpPipeline
         /// </summary>
         private readonly int _perRetryIndex;
 
+        private readonly TimeSpan _networkTimeout;
```

Comment:
Note: with NetworkTimeout no longer managed by the pipeline by ResponseBodyPolicy, we move it to a central location on the HttpPipeline itself.